### DISCUSSION TITLE
Access github actions local port with public ip

### DISCUSSION THREAD

**Discussion by tapan2930** [community/community#26067] [Actions]
<p>Is it possible to host a server(temporarily) in github action VM instance which can be accessed with the help of ip and port name ?</p>
<p>Basically I can access the public ip of vm instance and port on which express app is running. All I need is a way to open that express app in a browser with the help of url.</p>
<p>Thanks in advance</p>

**Comment by mchirico** [comment, marked as answer]
<p>Yes.  It is possible to suspend a running action, then, access the running, action, container directly via NGROK.</p>
<p>For example, we’ll develop or troubleshoot a Github/Action with other team members. We’ll suspend the action, so that we can all login and investigate. To do this, we run the node code-server application, on the Action VM.</p>
<p>Here’s a video on the walk-through process… showing you how to use NGROK, for getting direct access.</p>
<p><a href="https://player.vimeo.com/video/449383747" class="onebox" target="_blank" rel="nofollow noopener">https://player.vimeo.com/video/449383747</a></p>
<p>Hope this helps</p>
